LayoutTests/inspector/view/basics.html

```diff
@@ -228,12 +228,20 @@
             InspectorTest.expectFalse(parent._dirty, "Parent detached from root view should not be dirty.");
             InspectorTest.expectFalse(child._dirty, "Child attached to detached parent view should not be dirty.");
 
+            InspectorTest.log("- Dirtying parent.");
+            parent.needsLayout();
+            InspectorTest.expectEqual(rootView._dirtyDescendantsCount, 0, "Root view should have 0 dirty descendants.");
+            InspectorTest.expectEqual(parent._dirtyDescendantsCount, 0, "Parent attached to root view should have 0 dirty descendants.");
+            InspectorTest.expectFalse(rootView._dirty, "Root view should not be dirty.");
+            InspectorTest.expectTrue(parent._dirty, "Parent detached from root view should be dirty.");
+            InspectorTest.expectFalse(child._dirty, "Child attached to parent view should not be dirty.");
+
             InspectorTest.log("- Dirtying child.");
             child.needsLayout();
             InspectorTest.expectEqual(rootView._dirtyDescendantsCount, 0, "Root view should have 0 dirty descendants.");
             InspectorTest.expectEqual(parent._dirtyDescendantsCount, 1, "Parent attached to root view should have 1 dirty descendants.");
             InspectorTest.expectFalse(rootView._dirty, "Root view should not be dirty.");
-            InspectorTest.expectFalse(parent._dirty, "Parent detached from root view should not be dirty.");
+            InspectorTest.expectTrue(parent._dirty, "Parent detached from root view should not be dirty.");
```

Comment:
Should the message here be updated as well?

Reply:
ah lol oops yes will fix :)